### PR TITLE
Issue#980

### PR DESCRIPTION
**Pull request by nitishksahu**
Test Specs

### CODE REVIEW

File: record/index.html.in [hunk 1]
```diff
@@ -56,7 +56,7 @@
...
                     <uib-accordion-heading>
                         <i class="pull-left glyphicon" style="margin-right: 5px;" ng-class="{'glyphicon-chevron-down': tableModels[$index].open, 'glyphicon-chevron-right': !tableModels[$index].open}"></i>
-                        <span ng-if="::relatedRef.displayname.isHTML" ng-bind-html="::relatedRef.displayname.value"></span><span ng-if="::!relatedRef.displayname.isHTML" ng-bind="::relatedRef.displayname.value"></span> ({{::tableModels[$index].rowValues.length}}{{::tableModels[$index].hasNext ? '+' : ''}})
+                        <span ng-if="::relatedRef.displayname.isHTML" ng-bind-html="::relatedRef.displayname.value"></span><span ng-if="::!relatedRef.displayname.isHTML" ng-bind="::relatedRef.displayname.value"></span> (Showing first {{::tableModels[$index].rowValues.length}}{{::tableModels[$index].hasNext ? '+' : ''}} {{::tableModels[$index].rowValues.length>1 ? ' results' : ' result'}})
```
Comment (jrchudy) [inline record/index.html.in]: It looks like you are adding 2 whitespaces between the number of results and the string `result`/`results`.

File: test/e2e/utils/record-helpers.js [hunk 2]
```diff
@@ -187,11 +187,14 @@ exports.testPresentation = function (tableParams) {
...
-                            expect(headings[i]).toBe(title + " (" + rowCount + "+)");
-                        } else {
+                            expect(headings[i]).toBe(title + " (Showing first " + rowCount + "+  results)");
+                        } else if(rowCount<=1){
                         // All other tables should not have the + at the end its heading
-                            expect(headings[i]).toBe(title + " (" + rowCount + ")");
+                            expect(headings[i]).toBe(title + " (Showing first " +rowCount+ "  result)");
                         }
+												else{
+													expect(headings[i]).toBe(title + " (Showing first " +rowCount+ "  results)");
```
Comment (jrchudy) [inline test/e2e/utils/record-helpers.js]: This `else` condition is checking the same exact case as the `displayName == tableParams.related_table_name...` condition above. This `if`/`else if`/`else` condition should be reduced to an `if`/`else` condition.

Comment (nitishksahu) [inline test/e2e/utils/record-helpers.js]: the first if is checking for the "+" character in the string whereas the else if and else condition are two scenarios where the count is greater than 1 then display "results" else display "result". If I merge the else part with the if condition then we can't differentiate between a case of" Showing 2+ results" and "Showing 2 results".

File: test/e2e/specs/default-config/record/related-table-link.spec.js [hunk 3]
```diff
@@ -8,9 +8,9 @@ var testParams = {
         operator: "="
     },
     tables_order: [
-        "accommodation_image (2+)",
-        "booking (6)",
-        "media (1)"
+        "accommodation_image (Showing first 2+  results)",
+        "booking (Showing first 6  results)",
+        "media (Showing first 1  result)"
```
Comment (jrchudy) [inline test/e2e/specs/default-config/record/related-table-link.spec.js]: These 3 expectations should be updated to have only 1 whitespace in them.

File: test/e2e/utils/record-helpers.js (hunk 2 above)
Comment (jrchudy) [inline test/e2e/utils/record-helpers.js]: You are correct. I missed that when I was looking at those conditions.

**Comment by jrchudy** [comment]
I pulled the changes to my local machine and ran the tests. I can verify that the tests still pass and can verify that your changes are working as we expect.

When you are ready, go ahead and merge this code.

**Pull request merged**